src/common/types/user.ts

```
@@ -8,3 +8,18 @@ export const searchUserByUinRequest = z.object({
 export const searchUserByUinResponse = z.object({
   email: z.email(),
 });
+
+export const batchResolveUserInfoRequest = z.object({
+  emails: z.array(z.email()).min(1)
```

ESLint

[error] Insert `,`

(prettier/prettier)